docs/usage.md

```diff
@@ -116,14 +117,21 @@ ChIPSampleID3,,AnalysisID3
 For single-sample peaking calling without a control sample, leave the second column blank
 (control sample name).
 
+### `--macsgsize`
+Effective genome size which is used for the option `--gsize` in MACS. Should be in the format "2.1e9". See [`conf/igenomes.config`](conf/igenomes.config) for the predefined values of all supported reference genomes.
```

Comment:
Is there some documentation that we can link to describing how people can calculate this for their own genomes? This would probably be better than linking to the default genome values.

Reply:
I have added more detailed descriptions in the doc. But I think we can still keep the default values because these values don't need to be very accurate. People will run into more trouble if there is no default value for their genome and MACS cannot run.

Reply:
Yes for sure - I’m mostly thinking of people using genomes not in this list. For example, if I have a random de-novo genome that I’ve built myself, how do I calculate the number.?